man/ocamlrun.m

```diff
@@ -119,56 +132,56 @@ in the OCAMLRUNPARAM environment variable (see below).
 and an optional multiplier. If the letter is followed by anything
 else, the corresponding option is set to 1. Unknown letters
 are ignored.
-The options are documented below; the
-last six correspond to the fields of the
+The options are documented below; the options
+.B a, i, l, m, M, n, o, O, s, v
```

Comment:
@damiendoligez - I think option `w` should be here too (`window_size` in the Gc control)?

Reply:
Thank you, David! 

Seems the option `w`'s description is the same with `window_size` in Gc control. Looking forward to @damiendoligez's confirmation.